### ISSUE TITLE
Adjust Hut Wood Rarity

### ISSUE THREAD

**Issue by Drullkus** [TeamTwilight/twilightforest#1294]
For Huts and Wells

First: Canopy, Spruce
Second: Twilight Oak, Oak
Third: Birch, Jungle, Mangrove, Dark Oak
Fourth: Magic woods

**Comment by saltyseadoggo** [comment]
Is acacia being removed from the huts entirely?

**Comment by GizmoTheMoonPig** [comment]
Yeah, we decided the color wasn't really a good fit for the huts

**Comment by saltyseadoggo** [comment]
Makes sense.
Also, why will dark oak be so much rarer than canopy when the two are quite similar in shade?
And did the team ever consider the 1.16 woods for them? Crimson in particular could possibly look decent

**Comment by GizmoTheMoonPig** [comment]
Dark oak is slightly lower just because it's darker than canopy or spruce, and we felt like those 2 fit best. As for nether woods, I don't think that they exactly fit in the TF first of all, and the colors are rather obnoxious and would severely clash with the hut style. Hope this answers your question 
Edit: oh I forgot we lowered dark oak to the 3rd tier. Honestly not too sure why we did but the builders on the team made that decision so I'm not even gonna question it.
